lib/e2e/src/lib.rs

```diff
@@ -1,12 +1,88 @@
-mod assert;
+mod assertions;
 mod deploy;
 mod environment;
 mod project;
 mod system;
 mod user;
 
-pub use assert::Assert;
+pub use assertions::{Assert, Emits};
 pub use deploy::deploy;
 pub use e2e_proc::test;
 pub use system::{provider, Provider, Signer};
 pub use user::User;
+
+/// This macro provides an easy way for broadcasting the transaction
```

Comment:
What do you think of changing the wording like this?

```suggestion
/// This macro provides a shorthand for broadcasting the transaction
```